core/partitioning/simpleDataPacker.go

```diff
@@ -0,0 +1,71 @@
+package partitioning
+
+import (
+	"github.com/ElrondNetwork/elrond-go/core"
+	"github.com/ElrondNetwork/elrond-go/marshal"
+)
+
+// SimpleDataPacker can split a large slice of byte slices in chunks <= maxPacketSize
+// If one element still exceeds maxPacketSize, it will be returned alone
+// It does the marshaling of the resulted (smaller) slice of byte slices
+type SimpleDataPacker struct {
+	marshalizer marshal.Marshalizer
+}
+
+// NewSimpleDataPacker creates a new SizeDataPacker instance
+func NewSimpleDataPacker(marshalizer marshal.Marshalizer) (*SimpleDataPacker, error) {
+	if marshalizer == nil || marshalizer.IsInterfaceNil() {
```

Comment:
use your core func of isNil :)

Reply:
Agree

Reply:
Not available yet. It is on feat/interceptors branch.